### PR TITLE
Enhance service binding client

### PR DESCRIPTION
**Pull request by dimityrmirchev**
**What this PR does / why we need it**:
This PR adds:
 - additional function called `get_service_binding` to the SBClient
 - additional fields `created_at` and `updated_at` to be returned when getting a service binding

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
/cc @MartinWeindel
 
**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: cfg_mgmt/btp_service_binding.py [hunk 1]
```diff
@@ -98,9 +118,21 @@ def get_service_bindings(self) -> list[ServiceBindingInfo]:
...
             name = item['name']
             instance_id = item['service_instance_id']
-            bindings.append(ServiceBindingInfo(id=id, name=name, service_instance_id=instance_id))
+            created_at = item['created_at']
+            updated_at = item['updated_at']
+            bindings.append(ServiceBindingInfo(
+                id=id, name=name, service_instance_id=instance_id, created_at=created_at, updated_at=updated_at))
         return bindings
 
+    def __response_json_to_sb_detailed_info(self, resp_json) -> ServiceBindingDetailedInfo:
```
Comment (ccwienk) [inline cfg_mgmt/btp_service_binding.py:127]: codestyle: "private" methods/functions should be marked w/ a single underscore. double-underscores are typically "reserved" for python-internals or -contracts 

consider using `dacite.from_dict` for deserialising `dict` -> `dataclass`

btw, using `dacite.config.Config`'s typehooks, you might even automatically deserialise timestamp-strings into `datetime.datetime` objects (if this is helpful/desirable), like so:


```
dacite.from_dict(
  data=resp_json,
  data_class=ServiceBindingDetailedInfo,
  config=dacite.config.Config(
    type_hooks={
     datetime.datetime: dateutil.parser.isoparse, # prefer dateutil.parser over python-default as the latter is buggy
   }
)
```

(for this to work, you would have to change the typehints for updated_at, created_at)

File: cfg_mgmt/btp_service_binding.py [hunk 2]
```diff
@@ -98,9 +118,21 @@ def get_service_bindings(self) -> list[ServiceBindingInfo]:
...
+        id = resp_json['id']
+        name = resp_json['name']
+        instance_id = resp_json['service_instance_id']
+        credentials = resp_json["credentials"]
+        created_at = resp_json['created_at']
+        updated_at = resp_json['updated_at']
+        return ServiceBindingDetailedInfo(id=id, name=name, service_instance_id=instance_id,
```
Comment (ccwienk) [inline cfg_mgmt/btp_service_binding.py:134]: codestyle: we prefer to have one argument per line

**Review by ccwienk — CHANGES_REQUESTED** [review]
some codestyle-nits, otherwise looks fine

**Comment by dimityrmirchev** [comment]
Thanks for the feedback, @ccwienk! I made the suggested changes. PTAL

**Review by ccwienk — APPROVED** [review]

**Pull request merged**